core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -165,6 +174,7 @@ object ConsumerPerformance {
 
     totalMessagesRead.set(messagesRead)
     totalBytesRead.set(bytesRead)
+    joinTime.set(joinEnd - joinStart)
```

Comment:
And further rebalances may be possible anyway depending on the state of the brokers.